app/matplotlib.ipynb

```diff
@@ -240,7 +263,21 @@
     "y1 = np.sin(x)\n",
     "y2 = np.cos(x)\n",
     "\n",
-    "# write your code here\n"
+    "fig, ax = plt.subplots(1, 2, figsize=(12, 6))\n",
+    "\n",
+    "fig.suptitle(\"Subplots\")\n",
+    "\n",
+    "ax[0].plot(x, y1, color=\"blue\")\n",
+    "ax[0].set_title(\"Sine Function\")\n",
+    "ax[0].set_xlabel(\"X Axis\")\n",
+    "ax[0].set_ylabel(\"Y Axis\")\n",
+    "\n",
+    "ax[1].plot(x, y2, color=\"yellow\")\n",
+    "ax[1].set_title(\"Cosine Function\")\n",
+    "ax[1].set_xlabel(\"X Axis\")\n",
+    "ax[1].set_ylabel(\"Y Axis\")\n",
+    "\n",
+    "plt.tight_layout(rect=[0, 0, 1, 0.95])\n"
```

Comment:
You should add `plt.show()` at the end of this cell to ensure the subplots are displayed in the notebook.